tests/protocol/test_hostplugin.py

```diff
@@ -244,6 +244,8 @@ def test_default_channel(self, patch_put, patch_upload, _):
         as part of status upload.
         """
         with self.create_mock_protocol() as wire_protocol:
+            wire.HostPluginProtocol.is_default_channel = False
```

Comment:
would it be worthwhile to add this as a mock inside `create_mock_protocol` since we want the state to be clean pretty much after every run?

Reply:
probably, but that requires additional changes in those tests, and probably many of the tests in this module have similar dependencies.

i fixed only the ones that broke